src/theme/components/modal.ts

```diff
@@ -17,18 +17,40 @@ const Modal = defineMultiStyleConfig({
       _dark: {
         bg: "dark.ui.bg.default",
       },
+      paddingBottom: "l",
```

Comment:
For consistency with the `header`, can this be added to the styles for the `footer`?